websites/C/CS Money/dist/metadata.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
+  "author": {
+    "name": "Kyrie",
+    "id": "368399721494216706"
+  },
+  "service": "CS Money",
+  "altnames": ["CS.Money"],
+  "description": {
+    "en": "A better way to trade CS:GO skins. Buy, sell, and trade skins easier and faster. All deals are secured with the highest level protection methods."
+  },
+  "url": ["cs.money","3d.cs.money","wiki.cs.money"],
```

Comment:
Use RegEx?

Reply:
The site only has 2 subdomains, is regEx required?